Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM golang:1.20-alpine3.18 AS builder
 
+ARG HAWTIO_ONLINE_VERSION=latest
+ARG HAWTIO_ONLINE_IMAGE_NAME=docker.io/hawtio/hawtio
```

Comment:
Let's move to `quay.io`. See #65

Reply:
> Let's move to `quay.io`. See #65

This will be done in a separate PR to keep the concerns separate.